.github/workflows/lint.yml

```
@@ -19,7 +19,14 @@ env:
   PYTHONUNBUFFERED: "1"
 
 jobs:
+  should-run:
+    if: github.event_name != 'push' || !contains(github.event.head_commit.message, ':bookmark: bump version')
```

actionlint (1.7.10)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)